apps/explorer/src/pages/address-result/AddressResult.tsx

```diff
@@ -25,37 +25,37 @@ function instanceOfDataType(object: any): object is DataType {
 }
 
 function AddressResult() {
-    const { id: addressID } = useParams();
+    const { id: addressId } = useParams();
 
-    if (addressID !== undefined) {
-        return (
-            <>
-                <div className="mt-5 mb-10">
-                    <PageHeader type="Address" title={addressID} />
-                </div>
-                <div>
-                    <h1>Owned Objects</h1>
-                    <div>
-                        <ErrorBoundary>
-                            <OwnedObjects id={addressID} byAddress />
-                        </ErrorBoundary>
-                    </div>
+    return (
+        <div className="space-y-12 xl:space-y-12">
```

Comment:
responsive css is the same as the default

Reply:
good catch